### PR TITLE
Wrapper test new

### PR DESCRIPTION
**Pull request by ajm01**

### CODE REVIEW

File: tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -7,7 +7,8 @@ Bundle-Version: 0.1.0
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Require-Bundle: org.junit.jupiter.api;bundle-version="[5.8.0,6.0.0)",
  org.eclipse.ui,
- org.hamcrest.library;bundle-version="[1.3.0,2.2)"
+ org.hamcrest.library;bundle-version="[1.3.0,2.2)",
+ io.openliberty.tools.eclipse.ui
```
Comment (scottkurz) [inline tests/META-INF/MANIFEST.MF:11]: We're mixing require-bundle and import-package?  Why not one or the other?

File: bundles/io.openliberty.tools.eclipse.ui/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -6,7 +6,8 @@ Bundle-SymbolicName: io.openliberty.tools.eclipse.ui;singleton:=true
 Bundle-Version: 0.1.0
 Bundle-Activator: io.openliberty.tools.eclipse.LibertyDevPlugin
 Export-Package: io.openliberty.tools.eclipse,
- io.openliberty.tools.eclipse.ui
+ io.openliberty.tools.eclipse.ui,
+ io.openliberty.tools.eclipse.utils
```
Comment (scottkurz) [inline bundles/io.openliberty.tools.eclipse.ui/META-INF/MANIFEST.MF:10]: Why this addition?

File: tests/META-INF/MANIFEST.MF (hunk 1 above)
Comment (ajm01) [reply tests/META-INF/MANIFEST.MF:11]: oh - i see...ok i will fix that, that seems to have gotten mixed up in the rename rebase i did

Comment (ajm01) [reply tests/META-INF/MANIFEST.MF:11]: got mixed up in the rename rebase ... i will fix

**Pull request merged**